.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy to Production environment
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the production branch
+  push:
+    branches: 
+    - production
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "deploy"
+  deploy:
+    # The GitHub environment that will be availble for reference
+    environment: "Production"
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - run: |
+          mkdir ~/.ssh
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/gh_actions_key
+          chmod 600 ~/.ssh/gh_actions_key
+          echo "${{ secrets.SSH_KNOWN_HOSTS }}" > ~/.ssh/known_hosts
+          scp -i ~/.ssh/gh_actions_key -P ${{ secrets.SSH_PORT }} -r ./ ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }}:/var/www/html/wp-content/themes/
```

Comment:
Would you mind adding a line to explaining what the flags for `scp` mean doing, and add a quick summary?

If there is  a chance to use the longer form flag names it would help our future selves, read this without needing to look them up all the time.